pkg/metadata/sandbox.go

```diff
@@ -115,11 +115,6 @@ func (s *sandboxStore) Get(sandboxID string) (*SandboxMetadata, error) {
 	if err != nil {
 		return nil, err
 	}
-	// Return nil without error if the corresponding metadata
-	// does not exist.
-	if data == nil {
-		return nil, nil
```

Comment:
yes.. it's nil.. just checking to make sure that's the error you wanted vs ErrNotExist